src/amp-story-player/amp-story-player-impl.js

```diff
@@ -648,7 +648,20 @@ export class AmpStoryPlayer {
    * @private
    */
   next_() {
-    if (this.currentIdx_ + 1 >= this.stories_.length) {
+    if (
+      this.element_.getAttribute('enable-circular-wrapping') != 'true' &&
```

Comment:
Do you need both conditions? I feel like there's a way to simplify it so that the second if statement can be removed or at least simplify the conditions.